src/traversal/printRelayOSSQuery.js

```diff
@@ -121,10 +121,8 @@ function printRoot(
     oneIndent + fieldName + children + newLine + '}';
 }
 
-function printMutation(
-  node: RelayQuery.Mutation,
-  printerState: PrinterState
-): string {
+function printOperation(node: RelayQuery.Operation, printerState: PrinterState): string {
+  const operationKind = node.getConcreteQueryNode().kind.toLowerCase();
```

Comment:
The text to print is an implementation detail, so how about passing the RelayQueryOperatiom object and add a ternary for mutation/subscription?